### PR TITLE
Nbo/parameters discard and edition prod 7862

### PR DESCRIPTION
**Pull request by Nikeleos**
Dev Review rules:
- 3 PR max
- at the second feedback, prefer meet instead of github conversation

Definition of Done:
- [ ] Testing
- [ ] No error logs in web console (except iFrame errors)
- [ ] Create or Update documentation (README file for users/integrators)
- [ ] Check the [test page](https://spaceport.cosmotech.com/confluence/pages/viewpage.action?spaceKey=QA&title=Azure+Sample+Webapp) in order to manually validate the feature 
- [ ] Clean code (regroup configuration variable, typos, no console.log...)

### CODE REVIEW

File: cypress/integration/brewery/CreateScenario-tests.spec.js [hunk 1]
```diff
@@ -24,21 +25,31 @@ const randomNmbr = (min, max) => {
...
   const otherScenarioName = SCENARIO_NAME.OTHER_SCENARIO;
 
   const stock = randomNmbr(BAR_PARAMETERS_RANGE.STOCK.MIN, BAR_PARAMETERS_RANGE.STOCK.MAX);
   const restock = randomNmbr(BAR_PARAMETERS_RANGE.RESTOCK.MIN, BAR_PARAMETERS_RANGE.RESTOCK.MAX);
   const waiters = randomNmbr(BAR_PARAMETERS_RANGE.WAITERS.MIN, BAR_PARAMETERS_RANGE.WAITERS.MAX);
 
-  const scenarioIdPattern = 's_.*';
+  const textField = randomStr(8);
+  const numberField = randomNmbr(BASIC_PARAMETERS_CONST.NUMBER.MIN, BASIC_PARAMETERS_CONST.NUMBER.MAX);
+  const enumField = randomEnum(BASIC_PARAMETERS_CONST.ENUM);
```
Comment (csm-thu) [inline cypress/integration/brewery/CreateScenario-tests.spec.js:46]: Can we name these `textValue`, `numberValue` and `enumValue` instead ?

File: cypress/integration/brewery/CreateScenario-tests.spec.js [hunk 2]
```diff
@@ -259,8 +270,124 @@ describe('Create scenario', () => {
...
+      scenarioCreatedName = req.response.body.name;
+      scenarioCreatedId = req.response.body.id;
+      cy.wrap(scenarioCreatedId).as('scenarioCreatedId');
+      expect(scenarioCreatedName).equal(scenarioWithBasicTypes);
+    });
+
+    cy.wait('@requestUpdateScenarioList').should((req) => {
+      const nameGet = req.response.body.find(obj => obj.id === scenarioCreatedId).name;
+      expect(nameGet).equal(scenarioCreatedName);
+      expect(nameGet).equal(scenarioWithBasicTypes);
```
Comment (csm-thu) [inline cypress/integration/brewery/CreateScenario-tests.spec.js:313]: If you already checked that `scenarioCreatedName` is equal to `scenarioWithBasicTypes`, you probably don't need this second check

File: src/components/ScenarioParameters/components/ScenarioParametersTabs.js [hunk 3]
```diff
@@ -92,6 +92,7 @@ const ScenarioParametersTabs = ({
             visibleTabs.map((tab, index) => (
               <Tab
                 key={index}
+                data-cy={tab.value}
```
Comment (csm-thu) [inline src/components/ScenarioParameters/components/ScenarioParametersTabs.js:95]: Could we add a prefix or suffix to have more info in the data-cy than just the parameters group name ? (e.g. `tab.value + '_tab'`)

**Review by csm-thu — APPROVED** [review]
Minor changes suggested, but it LGTM :+1:

**Review by csm-thu — APPROVED** [review]
LGTM ! :+1:

File: cypress/integration/brewery/CreateScenario-tests.spec.js [hunk 4]
```diff
@@ -24,21 +25,31 @@ const randomNmbr = (min, max) => {
   return Math.floor(Math.random() * (max - min) + min);
 };
 
+const randomEnum = (enumParam) => {
+  const rand = Math.floor(Math.random() * Object.keys(enumParam).length);
```
Comment (COSMO-VMI) [inline cypress/integration/brewery/CreateScenario-tests.spec.js:29]: What if `length === 0`?

**Review by COSMO-VMI — COMMENTED** [review]
Just a note on a borderline case.

**Review by csm-thu — APPROVED** [review]

File: cypress/integration/brewery/CreateScenario-tests.spec.js (hunk 4 above)
Comment (Nikeleos) [reply cypress/integration/brewery/CreateScenario-tests.spec.js:29]: Function return undefined

**Pull request merged**